### DISCUSSION TITLE
Deployed Alexa Music On VPS

### DISCUSSION THREAD

**Discussion by TheTeamAlexa** [TheTeamAlexa/TheTeamAlexa#3] [Q&A] [answered]
How we can deploye Alexa Music on VPS?

**Comment by Team-Alisha** [comment, marked as answer]
FROM nikolaik/python-nodejs:python3.9-nodejs18
RUN apt-get update -y && apt-get upgrade -y \
    && apt-get install -y --no-install-recommends ffmpeg \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/*
COPY . /app/
WORKDIR /app/
RUN git clone https://github.com/TheTeamAlexa/AlexaMusic AlexaMusic
RUN cd AlexaMusic && pip3 install -U -r requirements.txt
CMD cd AlexaMusic && bash start
